doc/source/installation.rst

```diff
@@ -116,6 +131,39 @@ By running :program:`pbm version`, you can verify if |pbm| has been built correc
    
       export PATH=/percona-backup-mongodb/bin:$PATH
 
+.. _tarball:
+
+Download packages from Percona website
+========================================
+
+You can `download installation packages <https://www.percona.com/downloads/percona-backup-mongodb/>`_ specific for your operating system from Percona website and install them using ``dpkg`` (Debian and Ubuntu) or ``rpm`` (Red Hat Enterprise Linux and CentOS). However, you must make sure that all dependencies are satisfied.
+
+Alternatively, you can download and install |pbm| from binary tarballs.
+
+Install from binary tarball
+---------------------------
+
+Find the link to the binary tarballs under the *Generic Linux* menu item on `Percona website <https://www.percona.com/downloads/percona-backup-mongodb/>`_.
+
+1. Fetch the correct binary tarball. For example, if you are running Debian 10 (“buster”):
```

Comment:
There is only one tarball, so I think maybe there is no need to set an example with buster (otherwise there is an impression that there are different tarballs for different OS)

Reply:
Yes, I recall that about half a year ago we changed from multiple linux tarballs to just one. So no surprise if we find older documentation that suggests there are a few different tarballs that are compiled linked with assumptions of a several different linux distributions, but I believe we create only one universal-use set of binaries for each product, which is built with more static linking and will run on any supported linux distro.